charts/bandstand-web-service/templates/_deployment.yaml

```diff
@@ -58,10 +58,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: status.hostIP
-            {{- range $.Values.additionalEnvVars}}
-            - name: {{ .name }}
-              value: {{ .value }}
-            {{- end }}
+            {{- toYaml .Values.additionalEnvVars | nindent 10 }}
```

Comment:
is 10 correct?

Reply:
not 100% sure - going to try it and see, is easier to test with a published chart. The examples I've seen seem to assume the indenting from the values will be added on